src/main.cpp

```diff
@@ -21,28 +25,6 @@ class Demo {
 
 		srcScreenRect = { 0, 0, DMG_WIDTH, DMG_HEIGHT };
 		destScreenRect = { x, y, DMG_WIDTH * pixelScale, DMG_HEIGHT * pixelScale };
-		
-		// TODO: Do another pass on the implementation of debug mode
```

Comment:
I'll revisit debug mode when I do an accuracy pass, I want to implement it in a more organized way.